src/main/java/com/google/sps/data/PersistentUserRepository.java

```diff
@@ -209,11 +223,26 @@ public void addUserToDatabase(User user) {
             userObject.setBio(userBio);
         }
 
+        if(bioMap != null) {
+            userObject.setBioMap(bioMap);
+        }
+
         userEntities.add(userObject);
     }
     return userEntities;
   }
 
+  //returns the bio map for given user
+  public Map<String, Integer> getMapForUser(User user) {
```

Comment:
Is this method needed, given fetchUserEntities already deserializes the map?

Reply:
For some reason when I was testing, the fetchUserEntities wasn't returning an updated map, but when I used this method to directly access the map, it worked. I will try to look into why this happened